packages/web/src/components/CheckboxField/_CheckboxField.scss

```
@@ -0,0 +1,81 @@
+@use 'theme';
+@use '../../tools/accessibility';
+@use '../../tools/typography';
+
+.CheckboxField {
+    display: inline-flex;
+    margin: theme.$spacing 0;
```

Do we need to reset inline margin? 
```suggestion
    margin-block: theme.$spacing;
```

---

Fixed.